src/handlers/transactionRequests.js

```diff
@@ -15,6 +15,6 @@ module.exports = {
    */
   post: function (request, h) {
     transactionRequest.forwardTransactionRequest(request, Enum.endpoints.TRANSACTION_REQUEST_POST)
-    return h.response().code(201)
+    return h.response().code(202)
```

Comment:
I am feeling more so than ever that we should have a standard lib for HTTP based stuff, i.e:
- standard health check server @lewisdaly 
- standard HTTP error code enums

@lewisdaly @rmothilal let me know what you guys think about this idea.